src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,14 @@
-export const UserInfo = () => (
-  <>Put the user here</>
-);
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => {
+  const {
+    email,
+    name,
+  } = user;
```

Comment:
Looks a bit too much for just two short-name properties.
```suggestion
  const { email, name } = user;
```